.buildkite/hierarchies/pipeline.yml

```diff
@@ -50,22 +50,18 @@ steps:
 
       - label: "Clima: GPU ClimaCoupler dry HS"
         command:
-          - "echo $$JULIA_DEPOT_PATH"
-
-          - echo "--- Instantiate AMIP env"
-          - "julia --project=experiments/ClimaEarth/ -e 'using Pkg; Pkg.instantiate(;verbose=true)'"
-          - "julia --project=experiments/ClimaEarth/ -e 'using Pkg; Pkg.precompile()'"
-          - "julia --project=experiments/ClimaEarth/ -e 'using Pkg; Pkg.status()'"
-
-          - echo "--- Download artifacts"
-          - "julia --project=artifacts -e 'using Pkg; Pkg.instantiate(;verbose=true)'"
-          - "julia --project=artifacts -e 'using Pkg; Pkg.precompile()'"
-          - "julia --project=artifacts -e 'using Pkg; Pkg.status()'"
-          - "julia --project=artifacts artifacts/download_artifacts.jl"
-
-          - echo "--- Run simulation"
           - "julia --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/run_dry_held_suarez.jl"
         artifact_paths:  "dry_held_suarez/dry_held_suarez/clima_atmos/*"
+        agents:
+          queue: clima
```

Comment:
If everything in this pipeline will run on clima, you can also change `queue` in line 2 of this file to `clima` instead of `new-central`, then everything will run on clima by default

Reply:
Good point, but I set it up on new-central intentionally as we may need to run some of the sims on it  once we add all of them (like we do with the longruns), so we don't take up all of the processes on clima. 👀

Reply:
sounds good!